.travis.yml

```diff
@@ -12,6 +12,7 @@ cache:
 install: true
 
 script:
+  - find ./ -name "pom.xml" -type f -exec mvn xml:check-format -f {} \;
```

Comment:
I changed to use `mvn-build-all.sh` at 6eae2783576bc9cbc349efa051103723951aebe0 commit .